tools/rapids-download-wheels-from-github

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# A utility script that downloads wheel artifacts from Github Artifacts to a temporary directory and returns the path to the directory
+set -eo pipefail
+
+source rapids-prompt-local-repo-config
+
+# For legacy reasons, allow this script to be run without the pkg_type being the first arg
+pkg_name="$(rapids-package-name "wheel_python")"
```

Comment:
Do we care about these "legacy reasons" in a new wrapper script?  I think we should eliminate support for older patterns since this is a new pattern.